src/Control/Effect/FossaApiClient.hs

```diff
@@ -84,6 +85,7 @@ data FossaApiClientF a where
   GetApiOpts :: FossaApiClientF ApiOpts
   GetAttribution :: ProjectRevision -> ReportOutputFormat -> FossaApiClientF Text
   GetIssues :: ProjectRevision -> FossaApiClientF Issues
+  GetEndpointVersion :: FossaApiClientF (Maybe Text)
```

Comment:
We should this new endpoint to the `matchExpectation` definition in `Test.MockApi`